### ISSUE TITLE
Javascript stops working after adding bootstrap in rails 3.2.12 app

### ISSUE THREAD

**Issue by emclab** [twbs/bootstrap-sass#350]
We added bootstrap to our rails 3.2.12 app. The format is better. However we notice that all the js features, such as inserting text boxes for new data or insert a partial form are not working any more. There is no response after clicking the adding-new-fields link on the form. Also jquery data-picker stops working as well. 

Here is the application.js in rails app:

  //= require jquery
  //= require jquery_ujs
  //= require jquery-ui
  //= require_tree .
  //= require bootstrap

The bootstrap was added at the end and after jquery. 

Under views/layouts/application.html.erb, there is a line for jquery:

link href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8/themes/base/jquery-ui.css" rel="stylesheet" type="text/css"/ 

Here is our application.css.scss:

  $iconSpritePath: image-path('glyphicons-halflings.png');
  $iconWhiteSpritePath: image-path('glyphicons-halflings-white.png');
  $navbarBackground: #555;
  $navbarBackgroundHighlight: #888;
  $navbarText: #eee;
  $navbarLinkColor: #eee;

  @import "bootstrap";
  body { padding-top: 60px;  }
  @import "bootstrap-responsive";
  @import "simple_form.css.scss";

Is it compatibility issue of bootstrap? How to fix the problem?
